### ISSUE TITLE
No liveness or readyness probes 

### ISSUE THREAD

**Issue by colegatron** [andviro/grayproxy#10]
Hi Andrew,

Load Balancers on AWS EKS does not support UDP protocol what forces me to set up a graylog process on each node of the k8s cluster, which in turn is waste of cpu and ram
My idea is to replace the graylog processes with grayproxy because of its smaller footprint.

For Kubernetes know when the process is ready to get traffic and also know if it continues healthy it requires to run a readiness and liveness probes, be it run a command (exit code=0 success), check if a tcp socket is listening (if it opens=success) or check if an http port is listening (if 200>=statuscode<=400 is success)

Have you thought on adding that kind of probes to the process? I am going to write a Helm Chart to setup Grayproxy on K8S and would be a great addition.

**Comment by andviro** [comment]
sure, will do. Minor difficulty is to choose which TCP port we should use, since there could be arbitrary HTTP inputs.

**Comment by colegatron** [comment]
Thanks for answering

Since the idea is to use it inside a container and it has its own ip the
chosen port does not really matter, up to you. Maybe it is a good idea have
a default port but let the user specify it through an environment variable.

Btw, yesterday I was setting up the thing and I really missed a debug
config switch to see what the app was doing, because I am having troubles
with the communication steps until I reach the container.

**Comment by andviro** [comment]
yep, I know it. Sorry, having a lot on the plate right now, no time to do something about it. Will try to find some time in next couple of weeks.
